### DISCUSSION TITLE
How to disable automatic sorting for datatables

### DISCUSSION THREAD

**Discussion by reogle** [dropways/deskapp#64] [Q&A] [answered]
I'm using the datatables with export so I can print and export to PDF. However, the table keeps automatically sorting on the first column. How do I tell it to stop sorting automatically?

**Comment by TakheleTsabedze** [comment, marked as answer]
Add this class to the column you do not want to sort. class="table-plus datatable-nosort"
For example I do not want to sort the House Code then it would be <th class="table-plus datatable-nosort">HOUSE CODE</th>
